README.md

```diff
@@ -1,57 +1,104 @@
 # cio-sdk-tools
 
+A collection of tools to assist in the integration of the Customer.io mobile SDKs.
+
 ## 🚧 This tool is a work in progress. 
-For feedback or feature requests, [open an issue](https://github.com/customerio/cio-sdk-tools/issues/new). The tool aims to diagnose common scenarios when integrating the Customer.io mobile SDK in your mobile app. Keep in mind that advanced or custom implementations might require manual troubleshooting.
+For feedback or feature requests, [open an issue](https://github.com/customerio/cio-sdk-tools/issues/new).
+
+## Available Commands
+<details>
+   <summary>doctor: Diagnose common scenarios when integrating the Customer.io mobile SDK</summary>
+
+   ## What It Does
+   The tool assists in diagnosing and troubleshooting the Customer.io mobile SDK installations. It examines:
+
+   1. **Project Setup**: Recognizing your mobile framework, such as React Native.
+   2. **SDK Initialization**: Verification of the SDK's initiation within key project files based on our setup guide recommendations.
+   3. **Push Notification Setup**:
+      - Validation of the presence and correct embedding of Notification Service Extensions.
+      - Verification of deployment target versions to ensure compatibility with the CIO SDK.
+      - Examination of `AppDelegate` to ensure correct metrics tracking for push notifications.
+      - Checking entitlements for push notification capabilities and potential conflicts.
+   4. **Dependencies**:
+      - Validation against any conflicting libraries in `package.json` and `Podfile` for now.
+      - Consolidates and displays versions of key integrations like the Customer.io SDK in various configuration files.
```

Comment:
I am not sure this deep of detail is relevant or important to customers who are reading this README document. 

I liked the above sentence "The tool assists in diagnosing and troubleshooting the Customer.io mobile SDK installations." as explaining the doctor command. That might be enough detail to include?

Reply:
Oh, wait. Now that I look deeper at the diff of this file, you may not have added this section yourself. If that is correct, ignore this comment.

Reply:
yup just moved those